Sources/Purchasing/Purchases/Purchases.swift

```
@@ -1697,6 +1699,15 @@ private extension Purchases {
         }
     }
 
+    private func postPaywallEventsIfNeeded() {
+        guard #available(iOS 15.0, tvOS 15.0, macOS 12.0, watchOS 8.0, *),
+              let manager = self.paywallEventsManager else { return }
+
+        self.operationDispatcher.dispatchOnWorkerThread(delay: .long) {
+            _ = try? await manager.flushEvents(count: PaywallEventsManager.defaultEventFlushCount)
```

Ignoring errors because they're already logged internally.